graphics/include/gz/common/MeshManager.hh

```diff
@@ -280,6 +280,21 @@ namespace gz
           const Mesh *_m2, const int _operation,
           const gz::math::Pose3d &_offset = gz::math::Pose3d::Zero);
 
+      /// \brief Perform convex decomposition on a submesh.
+      /// The submesh is decomposed into multiple convex submeshes. The output
+      /// submeshes contain vertices and indices but texture coordinates
+      /// not preserved.
+      /// \param[in] _subMesh Input submesh to decompose.
+      /// \param[in] _maxConvexHulls Maximum number of convex hull submeshes to
+      /// produce.
+      /// \param[in] _voxelResolution Voxel resolution to use. Higher value
+      /// produces more accurate shapes.
+      /// \return A vector of decomposed submeshes.
+      public: std::vector<SubMesh> ConvexDecomposition(
+                  const common::SubMesh *_subMesh,
```

Comment:
Any reason for using a pointer there instead of a const ref? IMO, it would be best for the caller to be responsible for checking that the submesh is not a nullptr.

Reply:
mainly to be consistent with other functions in MeshManager which all take mesh pointers. I updated this to const ref. [a6c5a2f](https://github.com/gazebosim/gz-common/pull/585/commits/a6c5a2f6c565e4ec636c1f492631312b470a5696)